sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -746,8 +757,12 @@ int64_t CurlSession::ResponseBufferParser::Parse(
         }
         else if (this->state == ResponseParserState::Headers)
         {
-          // Add header. TODO: Do toLower so all headers are lowerCase
-          this->m_response->AddHeader(this->m_internalBuffer);
+          if (!this->m_response->AddHeader(this->m_internalBuffer))
```

Comment:
do we have to use `this->`? I thought we use `m_` to avoid using `this->`.

Reply:
We don't have guideline for this.
I like using `this->` personally, I would like to remove the `m_`  :D

Reply:
> We don't have guideline for this.

We probably should. Following .NET, we don't use the `this` and VS IDE even greys it out/suggests removing it. I prefer not having extra/redundant pieces of code that we don't truly need.